apps/fabric-website/src/pages/Controls/ButtonPage/docs/mac/ButtonOverview.md

```diff
@@ -0,0 +1,22 @@
+Buttons are one of the core controls that make an app feel native to the platform it's on. It’s important to respect the platform's paradigms in order to help the user feel at home on macOS and keep the experience quality high.
+
+Buttons are a standard control that users can interact with to take actions within your application. The FluentUI button for macOS adds system appropriate hover effects when your mouse rests over it.
```

Comment:
Has this text been reviewed? Each of these paragraphs feels like an overview/intro on its own, not something that makes sense to have in succession.

Reply:
Sorta. We have an internal copy of the markdown files in fluentui-apple, that was reviewed when I checked in the button control to begin with (https://github.com/microsoft/fluentui-apple/pull/67#discussion_r446277701)

In this case, the first paragraph is a bunch of generic text I saw for other FluentUI Buttons, and the second paragraph I typed myself after getting feedback it didn't say anything specific about what the macOS FluentUI button does.

Reply:
Ah okay. Looking at places where that text appears for other platforms, I agree it's a bit strange to use it (on those platforms as well) as the entire overview. Also, it seems more like guidance to those *implementing* the Fluent Button on a given platform, not guidance for developers *using* it--and the website is supposed to be for developers *using* our controls. (This is a repeated problem.)

For comparison, this is the latest version of the button overview for web, which has been reviewed by writers (though unfortunately is not live yet):
> Buttons give people a way to trigger an action. They're typically found in forms, dialog panels, and dialogs. Some buttons are specialized for particular tasks, such as navigation, repeated actions, or presenting menus.

This is probably a more appropriate overview (aside from the "dialog panels" bit which is rather silly--not sure how that got in), and you could reuse it for macOS with minor modifications.

Reply:
Updated the wording. @ecraig12345 would you or someone else be able to look at it?